database/seed.js

```diff
@@ -1,4 +1,33 @@
+const db = require('./index.js');
 const data = require('../database/dummy_data.js');
 
-data.createAuthor();
-data.createAuthor();
+let authorCount = 0;
+let bookCount = 0;
+
+let createAuthor = ()=> {
+    db.addAuthor({name: data.name(), details: data.details()}).then(() => {
+      authorCount++;
+      if(authorCount < 100) {
+        createAuthor()
+    }
+  })
+};
+
+let createBook = ()=> {
+    db.addBook({title: data.title(), description: data.description(), published_year: data.year({min:1920, max:2019})}).then(() => {
+      bookCount++;
+      if(bookCount < 100) {
+      createBook();
+    }
+  })
+}
+
+createAuthor();
+createBook();
+
+
+
```

Comment:
can I see the schema set up? don't see it in the branch

Reply:
I had it set to be ignored in error, i've pushed it up so you can see it now. thanks!